src/web/mod.rs

```diff
@@ -50,21 +58,23 @@ pub fn run(soup: Blender) -> HttpResult<Listening> {
     };
 
     for (path, ep) in ins.into_iter() {
-        use std::sync::Mutex;
-        let put = Mutex::new(ep.f);
+        // use std::sync::Mutex;
```

Comment:
Why would you rather have the import at the top? It's fine, I guess, but then you should remove this line entirely.